lib/services/user_service.dart

```diff
@@ -47,13 +50,11 @@ class UserService {
     required bool newsLetterSignup,
   }) async {
     // TODO Handle news letter
-    final response =
-        await _causeServiceClient.getMeApi().meProfilePartialUpdate(
-              patchedUserProfile:
-                  Api.PatchedUserProfile((profile) => profile..name = name),
-            );
-    _currentUser = UserProfile.fromApiModel(response.data!);
-    _currentUserStreamController.add(_currentUser);
+    await _causeServiceClient.getMeApi().meProfilePartialUpdate(
+          patchedUserProfileUpdate:
+              Api.PatchedUserProfileUpdate((profile) => profile..name = name),
+        );
+    await fetchUser();
```

Comment:
This is pretty sad. In an ideal world we would have the same return type for update and get but right now the API doesn't do that. For now Ive just set it so we refetch the user after update but in the future ill work out how to use django